site/cds_rdm/inspire_harvester/transform_entry.py

```diff
@@ -64,6 +68,8 @@ def build(self):
             "access": self._access(),
         }
 
+        if record.get("pids"):
+            rdm_record["pids"] = record["pids"]
```

Comment:
are you sure we can assign the pids directly? do they have the same structre in inspire?